src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -89,16 +86,7 @@ public void sendBuildStaredNotification(boolean isFromPrebuild) {
 
         for (Webhook webhook : property.getWebhooks()) {
             if (webhook.isStartNotification()) {
-                //         listener.getLogger().println(String.format("Notifying webhook '%s'", webhook));
-                try {
-                    HttpWorker worker = new HttpWorker(run.getEnvironment(listener).expand(webhook.getUrl()), gson.toJson(card), webhook.getTimeout(), 3, listener.getLogger());
-                    executorService.submit(worker);
-                } catch (Throwable error) {
-                    error.printStackTrace(listener.error(String.format("Failed to notify webhook '%s'", webhook)));
-                    listener.getLogger().println(String.format("Failed to notify webhook '%s' - %s: %s", webhook, error.getClass().getName(), error.getMessage()));
-                }
-            } else {
-                //    listener.getLogger().println(String.format("No need to notify webhook '%s'", webhook));
+                executeWorker(webhook, card);
```

Comment:
this code was repeated 3 times. I think that even loop for can be moved into this method but I need to think over one more time